ironfish/src/wallet/wallet.test.ts

```diff
@@ -315,9 +315,21 @@ describe('Accounts', () => {
   })
 
   describe('scanTransactions', () => {
+    it('should not rescan when wallet is not started', async () => {
+      const { node } = nodeTest
```

Comment:
I think you can create a new `nodeTest` in these unit tests to be a bit cleaner
```suggestion
      const nodeTest = createNodeTest(false, { config: { enableWallet: false } })
      const { node } = nodeTest
```

Alternatively, you can stop the wallet

Reply:
that is cleaner, changed